lens/lens_format.ml

```diff
@@ -12,5 +14,7 @@ let pp_newline_list f b v = pp_print_list ~pp_sep:pp_print_newline f b v
 
 let pp_padded ~length fmt f v =
   let str = Format.asprintf "%a" fmt v in
-  let count = Pervasives.max 0 @@ length - String.length str in
+  let count = Pervasives.max 0 @@ (length - String.length str) in
```

Comment:
I bet this one was fun to debug :)

Reply:
the `@@` is equivalent to the Haskell $, and should precisely be there to avoid the brackets. The brackets were added by ocamlformat, and I'm slightly confused why... I've removed the @@ for now.